showcase/app/routes/page-components/advanced-table.ts

```diff
@@ -6,14 +6,17 @@
 import Route from '@ember/routing/route';
 
 import { DENSITIES } from '@hashicorp/design-system-components/components/hds/advanced-table/index';
-import type {
-  HdsBadgeColors,
-  HdsBadgeTypes,
-} from '@hashicorp/design-system-components/components/hds/badge/types';
-import type { HdsIconSignature } from '@hashicorp/design-system-components/components/hds/icon/index';
 
 import type { ModelFrom } from 'showcase/utils/ModelFromRoute';
 
+import clusterData from 'showcase/mocks/cluster-data';
```

Comment:
```suggestion
import clusters from 'showcase/mocks/cluster-data';
```
[Issue] It looks like all the import names in the adv table, table, and pagination files are slightly off. Not sure how it's working with the current names.

Reply:
its working because it is the default export, you can import it with any name. I'll update tho for consistency.